src/components/feed/NodeDetails/NodeDetails.tsx

```diff
@@ -352,7 +352,7 @@ function getCommand(
     }
   }
 
-  let command = `$> docker run --rm -v $(pwd)/in:/incoming -v $(pwd)/out:/outgoing \\\n${dock_image} ${selfexec} `;
+  let command = `$> apptainer oci delete ${dock_image} & apptainer exec --bind $(pwd)/in:/incoming,$(pwd)/out:/outgoing ${dock_image} ${selfexec}`;
```

Comment:
The usage of `--bind` looks correct. Though I have a preference for `$PWD` instead of `$(pwd)`, would you change that?

Reply:
Yes, sure